terraform/gcp/projects/qcl.tfvars

```diff
@@ -0,0 +1,84 @@
+prefix                 = "qcl"
+project_id             = "qcl-hub"
+
+zone                   = "europe-west1-d"
+region                 = "europe-west1"
+
+core_node_machine_type = "n2-highmem-2"
+enable_network_policy  = true
+
+# Setup a filestore for in-cluster NFS
+enable_filestore       = true
+filestore_capacity_gb  = 2048
+
+user_buckets = {
+  "scratch-staging": {
+    "delete_after": 7
+  },
+  "scratch": {
+    "delete_after": 7
+  }
+}
+
+notebook_nodes = {
+  "medium" : {
+    min : 0,
+    max : 100,
+    machine_type : "n2-highmem-4",
+    labels: {},
+    gpu: {
+      enabled: false,
+      type: "",
+      count: 0
+    }
+  },
+  "large" : {
+    min : 0,
+    max : 100,
+    machine_type : "n2-highmem-16",
+    labels: {},
+    gpu: {
+      enabled: false,
+      type: "",
+      count: 0
+    }
+  },
```

Comment:
I've called these `small` and `medium` rather than `medium` and `large` in the profile list I'm trying advocating we standardize a bit around. That includes highmem 4 / 16 / 64 nodes, that the profile list refers to as small / medium / large.

To avoid confusion it would be good if these two are either named technically specific like `n2-highmem-4` like the machine or aligns with the user facing practice where `highmem-4` maps to `small` and `highmem-16` maps to `medium`.

This is all based on me trying to [nudge towards standarizing abit around specifically highmem 4 / 16 / 64 nodes](https://github.com/2i2c-org/infrastructure/issues/2121), but the legacy configuration has called small -> huge a range between 2 and 24 CPU i believe.

Reply:
Will using n2-highmem* and n2-highcpu* names endup in the profile list automatically and thus potentially suffer from the prefix matching issue seen in kubespawner?

Reply:
I think this name only influences the node pool name, which influences what you see when you do `kubectl get node`, as well as what users may see in grafana dashboards --- but the profile list is manually configured against labels, which are provided based on machine types.